homeassistant/components/switcher_kis/light.py

```diff
@@ -87,8 +95,8 @@ def is_on(self) -> bool:
         if self.control_result is not None:
             return self.control_result
 
-        data = cast(SwitcherSingleShutterDualLight, self.coordinator.data)
-        return bool(data.lights[self._light_id] == DeviceState.ON)
+        data = cast(SwitcherDualShutterSingleLight, self.coordinator.data)
+        return bool(data.lights == DeviceState.ON)
```

Comment:
Please move this to `SwitcherSingleLightEntity` class

Reply:
It is done.